noir-projects/mock-protocol-circuits/crates/app-creator/src/main.nr

```diff
@@ -0,0 +1,8 @@
+use dep::mock_types::{AppPublicInputs, MAX_COMMITMENTS_PER_CALL};
```

Comment:
Would you mind adding a couple of comments to these mock app methods indicating what is the spirit of the functionality being mocked and what is actually being done?